app/lib/constants/settings/general.rb

```diff
@@ -36,11 +36,13 @@ module General
           placeholder: "a secure token"
         },
         logo_png: {
-          description: "Used as a fallback to the SVG. Recommended minimum of 512x512px",
+          description: "Used as a secondary logo in places like the sign in modals, onboarding, apple touch icons. " \
+                       "Recommended minimum of 512x512px",
           placeholder: IMAGE_PLACEHOLDER
         },
         logo_svg: {
-          description: "Used as the SVG logo of the community",
+          description: "Currently being used on the upper left hand corner of your Forem, " \
+                       "however it will be deprecated soon in favour of Logo above.",
```

Comment:
Wording (non-blocking)

```suggestion
                       "however it will be deprecated soon in favour of the Logo filed above.",
```